app.py

```
@@ -74,9 +101,10 @@
 # ==========================================
 def server(input, output, session: Session):
     logger.info("📱 Shiny app session started")
+    logger.info(f"💾 Cache stats: {COMPUTATION_CACHE.get_stats()}")
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)